opentelemetry-instrumentation/src/opentelemetry/instrumentation/_blobupload/api/content_type.py

```diff
@@ -0,0 +1,28 @@
+"""Provides utilities for automatic content-type detection."""
+
+
+# Helper used to handle the possibility of optional 'magic' dependency
+# being unavailable for guessing the MIME type of raw bytes.
+class _FallBackModule(object):
```

Comment:
Why do you inherit object? https://stackoverflow.com/a/45062077/63621

Reply:
Removed.